src/ClusterManager/job_launcher.py

```diff
@@ -648,19 +643,27 @@ def submit_job_impl(self, job):
             jobMeta["LaunchCMD"] = pods[0].spec.containers[0].command
 
             jobMetaStr = base64.b64encode(json.dumps(jobMeta))
-            dataHandler.UpdateJobTextField(job_object.job_id, "jobMeta", jobMetaStr)
+
+            fields = {}
```

Comment:
Since we are talking about perf here, let's squeeze the tiny bit everywhere if possible. https://doughellmann.com/blog/2012/11/12/the-performance-impact-of-using-dict-instead-of-in-cpython-2-7-2/